aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteModelTree.java

```diff
@@ -45,6 +41,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 /**
  * Utility class to help traverse a tree of models by relationship.
```

Comment:
We missed getting a `SQLiteModelTreeTest.java` when this class was first introduced, but let's please get one started.

Reply:
added a test!